README.md

```diff
@@ -5,7 +5,7 @@
 
 A collection of lints to catch common mistakes and improve your [Rust](https://github.com/rust-lang/rust) code.
 
-[There are over 650 lints included in this crate!](https://rust-lang.github.io/rust-clippy/master/index.html)
+[There are over 700 lints included in this crate!](https://rust-lang.github.io/rust-clippy/master/index.html)
```

Comment:
🎉

Looks like this is what caused CI to fail, it wanted this updated

Reply:
I'm quite surprised that our CI has this check.

But, did we surpass the 700 milestone with this? 🎉